src/components/screens/home-screen/home-screen.tsx

```diff
@@ -119,6 +125,20 @@ class HomeScreen extends React.Component<Props, State> {
     );
   }
 
+  /**
+   * Method for greeting message
+   */
+  private greeting = () =>{
+    const hour = new Date().getHours();
+    if(hour >= this.state.moriningHour && hour < this.state.noonHour){
+      return strings.welcome.goodMorning;
+    }else if (hour >= this.state.noonHour && hour < this.state.eveningHour){
+      return strings.welcome.goodAfternoon;
+    }else{
+      return strings.welcome.goodEvening;
+    }
```

Comment:
```suggestion
    if (hour >= moriningHour && hour < noonHour) {
      return strings.welcome.goodMorning;
    }
    else if (hour >= noonHour && hour < eveningHour) {
      return strings.welcome.goodAfternoon;
    }
    else {
      return strings.welcome.goodEvening;
    }
```
Correct formatting and spaces